docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm

```diff
@@ -182,7 +183,6 @@ <h2>Popup Methods</h2>
    {
       tau.closePopup();
    });
-&lt;/script&gt;
 </pre></pre>
```

Comment:
Would you please check this \</pre>\</pre>?
It looks like it should be \</pre> to me.

Reply:
Fixed